### ISSUE TITLE
Create VCard from JSON

### ISSUE THREAD

**Issue by bkr32** [DanielHabenicht/ngx-vcard#558]
**Is your feature request related to a problem? Please describe.**
ngx-vcard should expose a function to download a vcard via a function.

**Describe the solution you'd like**
a function that consumes JSON file structured correctly and downloads a vcard

**Comment by DanielHabenicht** [comment]
Sry, but can you rephrase your request and provide an example. 
As far as I can see that is exactly what this package supplies: https://github.com/DanielHabenicht/ngx-vcard#use-as-formatter

**Comment by bkr32** [comment]
Apologizes for the lack of context.

Let's say you have a table and want to download a specified contact, you'd pass it's data with a click like

``` 
<button click='download(data)' />

download(data){
Vcard ={
name:{
firstName:data.FirstName
}

}
}

PossibleVcardDownloadFunction(Vcard)

```
Where `PossibleVcardDownloadFunction` is the exposed function from the package, I could meanwhile use the aforementioned method of creating it as string and downloading using file-saver

**Comment by DanielHabenicht** [comment]
Ahh okay, so you want be to expose this function:
https://github.com/DanielHabenicht/ngx-vcard/blob/1117956e673ce9980b03a9e352a2d0d22518ad1e/projects/ngx-vcard/src/lib/download-vcard.directive.ts#L25-L40

**Comment by bkr32** [comment]
pretty much, currently the download function is tied to the html component as a directive and consumes the `vcard` set onInit, trying to set and download after this point results in a `version undefined` error

**Comment by DanielHabenicht** [comment]
Can you make a reproduction of this bug and open another issue, it seems like it should not happen. 

I will add an update to expose the function.

**Comment by bkr32** [comment]
will do so shortly
